src/style.css

```diff
@@ -0,0 +1,100 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  box-sizing: border-box;
+  font-family: "Avenir", sans-serif;
+}
+
+.form {
+  margin-top: 20px;
+  position: relative;
```

Comment:
Input will broke if u add some element to top of the form, let's add relative to the input wrapper